### PR TITLE
[R4R] update api && example

### PR DESCRIPTION
**Pull request by dean65**
The DR update new feature of price calculations :
1. Support for new price calculations.
2. Add example of sending bundle with deposit coinbase.

The readme and example changed accordingly.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -98,6 +98,9 @@ to each other, the second transaction is allowed to be failed,
 and the bundle should be verified on chain during [now+20 second, now+80 second].
 This case shows you how to interact with smart contract through direct-route,
 and how to control the timing to be verified.
+4. `sendBNBByBundleWithDepositCoinbaseDemo()`. In this case, we use two different 
+accounts to send BNB to each other, and including deposit BNB to coinbase to increase *Bundle Price* the two transactions should be all 
+successful or all failed.
```
Comment (realuncle) [inline readme.md:112]: Can we past the deployed deposit contract address on mainnet in readme?

**Review by realuncle — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (j75689) [reply readme.md:112]: sure

Comment (j75689) [reply readme.md:112]: updated, please check

Comment (dean65) [reply readme.md:112]: sure

Comment (dean65) [reply readme.md:112]: updated, please check

**Pull request merged**